collect_app/build.gradle

```diff
@@ -252,7 +252,7 @@ dependencies {
     implementation "com.arcao:slf4j-timber:3.1@aar"
 
     implementation "com.google.zxing:core:3.3.0" //updating to 3.4.0 requires minSdkVersion >=24 https://github.com/zxing/zxing/releases/tag/zxing-3.4.0
-    implementation "com.journeyapps:zxing-android-embedded:3.5.0" //updating would require minSdkVersion >=24
+    implementation "com.journeyapps:zxing-android-embedded:3.6.0" //updating would require minSdkVersion >=24
```

Comment:
I turned out that 3.5.0 was not the last version we can use, 3.6.0 is, fortunately.